.github/workflows/main.yml

```diff
@@ -0,0 +1,22 @@
+name: theartofeducation.github.io
```

Comment:
We need to rename this file or the main workflow on the template-repo. They are both named main.

The one that comes in from the template will contain workflows that every project will use, like the Markdown linter. This one is specific to the project.

Thoughts?

Reply:
In my mind, main is the task the runs on every push, like linting and testing. In JS we would test with jest, but with python we would test with pytest. Each repo will need specific tasks for each kind of project.

Reply:
We decided on manual intervention for that.